Not needed since not in a flatpak where we need to ensure glibc compat,
and we use musl anyways
---
--- a/agent/meson.build
+++ b/agent/meson.build
@@ -42,18 +42,6 @@ ptyxis_agent_c_args = [
   '-DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_@0@_@1@'.format(PTYXIS_AGENT_MIN_GLIB_MAJOR, PTYXIS_AGENT_MIN_GLIB_MINOR),
 ]
 
-libc_compat = true
-if target_machine.cpu_family() == 'x86_64'
-  ptyxis_agent_c_args += ['-include', 'x86_64/force_link_glibc_2.17.h']
-else
-  libc_compat = false
-endif
-
-if libc_compat
-  config_h.set10('LIBC_COMPAT', libc_compat)
-  ptyxis_agent_link_args += ['-Wl,--wrap=__libc_start_main']
-endif
-
 executable('ptyxis-agent', ptyxis_agent_sources + ptyxis_agent_ipc,
          dependencies: ptyxis_agent_deps,
               install: true,
